src/main/java/school/lemon/changerequest/java/io/hw2/Demo.java

```diff
@@ -0,0 +1,43 @@
+package school.lemon.changerequest.java.io.hw2;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.FileAlreadyExistsException;
+
+/**
+ * Created by lera on 22.02.17.
+ */
+public class Demo {
```

Comment:
Please add some `.txt` file to your repo with results of your testing.
For example, you can write your output from System.out directly to some file.